cli/src/commit_templater.rs

```diff
@@ -1394,14 +1394,17 @@ fn builtin_tree_diff_methods<'repo>() -> CommitTemplateBuildMethodFnMap<'repo, T
             let path_converter = language.path_converter;
             let template = self_property
                 .map(move |diff| {
+                    let from_tree = diff.from_tree.clone();
                     let to_tree = diff.to_tree.clone();
-                    diff.into_formatted(move |formatter, _store, tree_diff| {
+                    let matcher = diff.matcher.clone();
+                    diff.into_formatted(move |formatter, _store, _tree_diff| {
```

Comment:
nit: can you leave a TODO comment?
`// TODO: don't pass separate copies of from_tree/to_tree/matcher`

Reply:
Done